v4/src/redux/reducers/graphlib.js

```diff
@@ -1,10 +1,12 @@
 import { sha256 } from 'js-sha256';
 import { NTYPE } from '../../types';
+// import { objToCid, verifyNodeShape } from 'ipfs-cytoscape';
 
 const incomers = (node) => node.incomers((el) => el.isNode());
 const outgoers = (node) => node.outgoers((el) => el.isNode());
 
 function nodeId(props, includeSource = true) {
+  // call objToCid synchronously
```

Comment:
I couldn't figure out a smart way to do this. Maybe @rianadon @susan-lin you can take a look? I want to call `objToCid` and return the result of that, but it's async due to the way it's implemented in the library I'm using. I think we'll have to write this into redux, or use redux-saga or something. Thoughts?